layouts/partials/tw/thumb-v-item.html

```diff
@@ -9,11 +9,12 @@
       -}}
     {{ end }}
   </div>
-
-  <div>
-    {{ partial "tw/image-block.html" .imgOpts }}
-  </div>
-  <div class="group mt-6">
+  {{- if ne .showImg false }}
```

Comment:
Is there a reason to do `if ne .showImg false` instead of `if .showImg`? Do we want it to default to shown? Usually it's better to make it so the defaults are false, like `if not .hideImage`, because things that aren't assigned a value are falsey.

Reply:
You can do `isset . "showImg"` if you really need to, but it's a little awkward: https://gohugo.io/functions/isset/

Reply:
I removed this. It was a remnant of previous design requests. https://github.com/spotlightpa/poor-richard/pull/356/commits/3c363bcd7647a33296bac34a63b8e4c815b764b5